hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -726,6 +757,11 @@ static int xpadneo_input_mapping(struct hid_device *hdev, struct hid_input *hi,
 {
 	int i = 0;
 
+	/* XBE2 reports a full keyboard, which we don't need */
+	if ((usage->hid & HID_USAGE_PAGE) == HID_UP_KEYBOARD) {
+		return MAP_IGNORE;
```

Comment:
According to Linux kernel code style guide, a single-instruction if body should not have curly braces around it.